atom/browser/javascript_environment.cc

```diff
@@ -27,8 +29,18 @@ bool JavascriptEnvironment::Initialize() {
     const char expose_debug_as[] = "--expose_debug_as=v8debug";
     v8::V8::SetFlagsFromString(expose_debug_as, sizeof(expose_debug_as) - 1);
   }
+
+  const std::string js_flags_switch = "js-flags";
+
+  if (cmd->HasSwitch(js_flags_switch)) {
```

Comment:
If you pass "js-flags" directly (like the way it's done with "debug-brk"), you can drop line 33 and 5 (and the extra empty lines 6, 34).